lib/js-yaml/dumper.js

```diff
@@ -738,7 +739,7 @@ function writeNode(state, level, object, block, compact, iskey) {
       if (block && (0 !== Object.keys(state.dump).length)) {
         writeBlockMapping(state, level, state.dump, compact);
         if (duplicate) {
-          state.dump = '&ref_' + duplicateIndex + (0 === level ? '\n' : '') + state.dump;
+          state.dump = '&ref_' + duplicateIndex + state.dump;
```

Comment:
Is this removed completely? Why? AFAIK this was intended for cases like the following:
``` yaml
a: &ref_1
  foo: bar
  answer: 42
b: *ref_1
```